.github/workflows/pr-check-shell.yaml

```diff
@@ -0,0 +1,15 @@
+name: Check Shell Code
+
+on:
```

Comment:
You can refer to other action yaml for this part, the push to main should trigger the action.

Reply:
Thanks, fixed